activesupport/lib/active_support/cache/file_store.rb

```diff
@@ -43,14 +43,33 @@ def cleanup(options = nil)
         end
       end
 
-      # Increments an already existing integer value that is stored in the cache.
-      # If the key is not found nothing is done.
+      # Increment a cached integer value in the store. Returns the updated value.
+      #
+      # If the key is unset, it will be set to +amount+:
```

Comment:
I think it's more logical to state it as:

```suggestion
      # If the key is unset, it starts from +0+:
```